### PR TITLE
Added blueprint check + missing spread dependency

### PR DESCRIPTION
**Pull request by dafortin**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* **Added** blueprint check
* **Added** missing dependency to `ember-spread`

### CODE REVIEW

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:
### No changes

**Comment by rox163** [comment]
:+1:

File: blueprints/ember-frost-file-picker/index.js [hunk 1]
```diff
@@ -1,19 +1,30 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-prop-types', target: '^3.0.0'},
+      {name: 'ember-hook', target: '^1.3.1'}
```
Comment (sglanzer) [inline blueprints/ember-frost-file-picker/index.js]: I'm sure you're on your way to do this, but hook and prop-types can be dropped

**Comment by rox163** [comment]
:+1:

**Pull request merged**